rep/db/db.py

```diff
@@ -35,6 +35,16 @@ def create_db(self):
                 FOREIGN KEY (sourceType) REFERENCES raw_vote_object_source_type(type), 
                 FOREIGN KEY (format) REFERENCES raw_vote_object_format(format)
             );
+            
+        CREATE TABLE IF NOT EXISTS processed_vote_results
```

Comment:
I've been going with singular table names `raw_vote_object`, `raw_vote_object_source_type`, `raw_vote_object_format`. Can we change this to  `processed_vote_result`, for consistency?

There's not _really_ a right or wrong, I just want to be consistent

Reply:
Sure, I'm into that